ports/metal-cpp/portfile.cmake

```
@@ -0,0 +1,32 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+vcpkg_download_distfile(
+    archive
+    URLS https://developer.apple.com/metal/cpp/files/metal-cpp_macOS12_iOS15.zip
+    FILENAME metal-cpp.zip
+    SHA512 dabb4109c7bf283288b5b3bd392892a7a52ad13b4d53a72a117a852f54b0a82871bec3e55c8493d8048365839bd1be37d72f872041f43db314622bb4a983921f
+)
+
+vcpkg_extract_source_archive_ex(
+  SOURCE_PATH ARCHIVE "${archive}"

```

```suggestion
  OUT_SOURCE_PATH ARCHIVE "${archive}"
```
Please use the latest method or standard usage. 
Reference： https://github.com/microsoft/vcpkg/blob/d369df7ecf194005eaca46f07368779cd486badd/docs/maintainers/vcpkg_extract_source_archive_ex.md